# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/silicit/cgilib/uplbuild.patch.cgiupl
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- libcgiupl-0.1/cgiupl.c.old	1998-05-12 04:55:54.000000000 +0200
+++ libcgiupl-0.1/cgiupl.c	2005-04-26 15:23:12.000000000 +0200
@@ -31,6 +31,7 @@
 #include <sys/stat.h>
 #include <string.h>
 #include <ctype.h>
+#include <errno.h>
 
 #include <cgiupl.h>
 
